src/TouristInfoCards.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
```

Comment:
This looks great. But it seems like there's some duplication here - the structure of each card is the same, and we are repeating that structure 3 times.
Using React, can you think of how you might be able to reduce that duplication? 😄

Reply:
Yes I think I can create a separate component "const Card" and pass all object info in there so I can add any other card with ease . thanks for letting me know :)